src/Depressurizer.Core/Helpers/Locations.cs

```diff
@@ -18,6 +18,10 @@ public static class File
             ///     Default database file location.
             /// </summary>
             public static string Database => "database.json";
+            /// <summary>
+            ///     Default database (gzipped) file location.
+            /// </summary>
+            public static string DatabaseGzip => "database.json.gz";
```

Comment:
Perhaps something similar to this construction to imply a relation between the two.

```suggestion
            public static string DatabaseGzip => Database + ".gz";
```